tasks/gene_typing/variant_detection/task_ivar_variant_call.wdl

```diff
@@ -71,18 +71,16 @@ task variant_call {
     cut -f 1-14 ~{samplename}.variants.tsv | awk '!seen[$0]++ {print}' > unique_variants.tsv
 
     # filter variants that pass fisher's exact test
-    grep "TRUE" unique_variants.tsv > passed_variants.tsv
+    grep "TRUE" unique_variants.tsv > passed_variants.tsv || touch passed_variants.tsv
 
     # calculate total number of variants
-    variants_num=$(cat passed_variants.tsv | wc -l)
-    if [ -z "$variants_num" ] ; then variants_num="0" ; fi
+    variants_num=$(wc -l < passed_variants.tsv || echo 0)
```

Comment:
this `|| echo 0` on this line is unnecessary since an empty file has 0 lines
ie, 
```
$ touch file
$ wc -l < file
0
```